diff --git a/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/FullScreenPlayerView.java b/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/FullScreenPlayerView.java
index f207ce8..e8acb63 100644
--- a/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/FullScreenPlayerView.java
+++ b/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/FullScreenPlayerView.java
@@ -41,11 +41,11 @@ public class FullScreenPlayerView extends Dialog {
         containerView.addView(exoPlayerView, generateDefaultLayoutParams());
 
         if (playerControlView != null) {
-            ImageButton imageButton = playerControlView.findViewById(com.brentvatne.react.R.id.exo_fullscreen);
-            imageButton.setImageResource(androidx.media3.ui.R.drawable.exo_icon_fullscreen_exit);
-            imageButton.setContentDescription(getContext().getString(androidx.media3.ui.R.string.exo_controls_fullscreen_exit_description));
+//            ImageButton imageButton = playerControlView.findViewById(com.brentvatne.react.R.id.exo_fullscreen);
+//            imageButton.setImageResource(androidx.media3.ui.R.drawable.exo_icon_fullscreen_exit);
+//            imageButton.setContentDescription(getContext().getString(androidx.media3.ui.R.string.exo_controls_fullscreen_exit_description));
             parent.removeView(playerControlView);
-            containerView.addView(playerControlView, generateDefaultLayoutParams());
+//            containerView.addView(playerControlView, generateDefaultLayoutParams());
         }
 
         super.onStart();
diff --git a/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/MyReceiver.java b/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/MyReceiver.java
new file mode 100644
index 0000000..41674ee
--- /dev/null
+++ b/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/MyReceiver.java
@@ -0,0 +1,29 @@
+package com.brentvatne.exoplayer;
+
+import static com.brentvatne.exoplayer.ReactExoplayerView.player;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+import java.util.Objects;
+
+public class MyReceiver extends BroadcastReceiver {
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        String action = intent.getStringExtra("action");
+        if (Objects.equals(action, "forward")) {
+            player.seekTo(player.getCurrentPosition() + 5000);
+        } else if (Objects.equals(action, "backward")) {
+            player.seekTo(player.getCurrentPosition() - 5000);
+        } else if (Objects.equals(action, "play")) {
+            if (player.isPlaying()) {
+                player.pause();
+            }else{
+                player.play();
+//                Toast.makeText(context.getApplicationContext(), "play",Toast.LENGTH_SHORT).show();
+
+            }
+        }
+    }
+}
diff --git a/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java b/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java
index bb09f1a..c06e3da 100644
--- a/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java
+++ b/node_modules/react-native-video/android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java
@@ -163,7 +163,7 @@ public class ReactExoplayerView extends FrameLayout implements
     private ImaAdsLoader adsLoader;
 
     private DataSource.Factory mediaDataSourceFactory;
-    private ExoPlayer player;
+    public static ExoPlayer player;
     private DefaultTrackSelector trackSelector;
     private boolean playerNeedsSource;
 
